src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -1062,6 +1062,45 @@ public void TestConfigPropertiesAreValid()
             configValidator.ValidateConfigProperties();
         }
 
+        /// <summary>
+        /// This method tests that config file is validated correctly and no exceptions are thrown.
+        /// </summary>
+        [TestMethod("Validates the complete config."), TestCategory(TestCategory.MSSQL)]
+        public async Task TestConfigIsValid()
```

Comment:
Why is this not added to `ConfigValidationUnitTests.cs`?

Reply:
this is an integration test and all the integration tests related to validations are in this class.